### PR TITLE
added space in range entries

### PR DESCRIPTION
**Pull request by ravi140222**
I wanted to make a change to functools file, but later realised that it was wrong, I wanted to undo that change, but there is no option to undo from GitHub, so only adding space change is there.
Plus this thing appears to be multiple times in the same file, should I add spaces to more places and create a PR?
[after some time]
I changed the functools file, to have wrap and unwrap, but I am not sure if it is the best way to mention them.
[after some time]
I changed the colorsys file, probably the way I wrote it is better than the existing.

### CODE REVIEW

**Comment by the-knights-who-say-ni** [comment]
Hello, and thanks for your contribution!

I'm a bot set up to make sure that the project can legally accept this contribution by verifying everyone involved has signed the [PSF contributor agreement](https://www.python.org/psf/contrib/contrib-form/) (CLA).

## Recognized GitHub username

We couldn't find a [bugs.python.org](https://bugs.python.org/) (b.p.o) account corresponding to the following GitHub usernames:

@ravi140222

This might be simply due to a missing "GitHub Name" entry in one's b.p.o account settings. This is necessary for legal reasons before we can look at this contribution. Please follow [the steps outlined in the CPython devguide](https://devguide.python.org/pullrequest/#licensing) to rectify this issue.




You can [check yourself](https://check-python-cla.herokuapp.com/) to see if the CLA has been received.

Thanks again for the contribution, we look forward to reviewing it!

File: Lib/colorsys.py [hunk 1]
```diff
@@ -151,16 +151,7 @@ def hsv_to_rgb(h, s, v):
...
-    if i == 3:
-        return p, q, v
-    if i == 4:
-        return t, p, v
-    if i == 5:
-        return v, p, q
+
+    l = [(v, t, p), (q, v, p), (p, v, t), (p, q, v), (t, p, v), (v, p, q)]
+    return l[i]
     # Cannot get here
```
Comment (arhadthedev) [inline Lib/colorsys.py]:
```suggestion
```

since there is nothing to remind of anymore.

Comment (ravi140222) [reply Lib/colorsys.py]: yes, changed it.

**Comment by ravi140222** [comment]
but is the conversion from if statement to list also cosmetic?
also, I changed the arguments in functools._unwrap_partial, is that also cosmetic change?

**Comment by sweeneyde** [comment]
Yes: unless there is a substantial objective improvement (for example speeding up of a frequently-used function or halving the number of lines of code) or you're making other changes to the same function/module, refactorings probably won't get accepted.

**Pull request closed without merging**